File: Frontend/src/app/model/doctor.ts
export class Doctor {
    doctorId: number;
    name: string;
    gender: string;
    phone: string;
    email: string;
    specialization: string;
    qualification: string;
    experience: string;
    location: string;
    availability: string;
    consultationFee: number;
    userId: number;
    hospitalId: number;
  
    constructor(
      doctorId: number,
      name: string,
      gender: string,
      phone: string,
      email: string,
      specialization: string,
      qualification: string,
      experience: string,
      location: string,
      availability: string,
      consultationFee: number,
      userId: number,
      hospitalId: number
    ) {
      this.doctorId = doctorId;
      this.name = name;
      this.gender = gender;
      this.phone = phone;
      this.email = email;
      this.specialization = specialization;
      this.qualification = qualification;
      this.experience = experience;
      this.location = location;
      this.availability = availability;
      this.consultationFee = consultationFee;
      this.userId = userId;
      this.hospitalId = hospitalId;
    }
  }
  

File: Backend/src/main/java/edu/icet/dto/Doctor.java
package edu.icet.dto;

import edu.icet.enums.Gender;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Doctor {
    private int doctorId;
    private String name;
    private Gender gender;
    private String phone;
    private String email;
    private String specialization;
    private String qualification;
    private String experience;
    private String location;
    private String availability;
    private double consultationFee;
    private int userId;
    private int hospitalId;
}
